python/kfserving/README.md

```diff
@@ -1 +1,39 @@
-TODO
\ No newline at end of file
+# KFServing
+
+KFServing is a unit of model serving, and every model type (XGBosst, PyTorch, Tensorflow etc.) extends KFServing. At a highlevel it defines a model, a server and storage mechanism for models.
+
+A model has essentially three functions:
+
+* Pre processing
+* Prediction 
+* Post processing
+
+A server on the other hand provides many funtionalities, inclding among others
```

Comment:
Can we make header sections for each of these handlers?

Reply:
with empty details for now? Or I can come back later when we are able to add more details around these functionalities